sklearn/linear_model/_coordinate_descent.py

```diff
@@ -461,14 +498,14 @@ def enet_path(
         List of alphas where to compute the models.
         If None alphas are set automatically.
 
-    precompute : 'auto', bool or array-like of shape \
-            (n_features, n_features), default='auto'
+    precompute : 'auto', bool or array-like of shape (n_features, n_features),
+        default='auto'
```

Comment:
When using the `raw` doc-string, the description for the `precompute` parameter renders incorrectly.

I’ve tried several "obvious fixes," including:

<img width="1032" alt="version_009_precompute" src="https://github.com/user-attachments/assets/980907ff-7ab4-4755-a912-7758aa3d37d3">

However, it seems we have to violate the [PEP-8 character limit](https://peps.python.org/pep-0008/#maximum-line-length) to get it right:

<img width="1049" alt="version_010" src="https://github.com/user-attachments/assets/4e14141c-f224-4712-aced-5658b18f9f64">

In particular, it appears this is an [ongoing issue with <code>numpydoc</code>](https://github.com/numpy/numpydoc/issues/87).

Reply:
Naive question, does using `\\` work?

Reply:
I previously tried this method, but `sphinx` didn’t render it correctly:

<img width="1034" alt="version_012" src="https://github.com/user-attachments/assets/6d7381e0-6b49-438e-b8b3-99b70986449a">